pkg/serverless/trace/trace.go

```diff
@@ -56,7 +54,7 @@ func (l *LoadConfig) Load() (*config.AgentConfig, error) {
 }
 
 // Start starts the agent
-func (s *ServerlessTraceAgent) Start(enabled bool, loadConfig Load, executionContext *executioncontext.ExecutionContext) {
+func (s *ServerlessTraceAgent) Start(enabled bool, loadConfig Load, lambdaSpanChan chan *pb.Span, coldStartSpanId uint64) {
```

Comment:
Might be nice to enforce channel directions for some of these span chan arguments. I forget the syntax specifically, but you can do something like

```go
func myThing(coolChan ->chan bool) { ... }
```

which then would fail to compile if you tried to pull something off the channel.

Reply:
Done!